go/apps/api/routes/v2_apis_list_keys/handler.go

```
@@ -225,252 +226,243 @@
+			HasMore: hasMore,
+		},
+	})
+}
 
-		if key.Key.Expires.Valid {
-			k.Expires = ptr.P(key.Key.Expires.Time.UnixMilli())
-		}
+// BuildKeyResponseData transforms internal key data into API response format.
+// Returns error on JSON unmarshaling failures to avoid silent data corruption.
+func (h *Handler) BuildKeyResponseData(keyData *db.KeyData, plaintext string) (openapi.KeyResponseData, error) {
+	response := openapi.KeyResponseData{
```

golangci-lint (2.2.2)

openapi.KeyResponseData is missing fields Credits, Expires, Identity, Meta, Name, Permissions, Plaintext, Ratelimits, Roles, UpdatedAt

(exhaustruct)